projects/components/src/table/header/table-header-cell-renderer.component.ts

```diff
@@ -44,11 +44,19 @@ import { TableColumnConfigExtended } from '../table.service';
             <div [style.min-width.px]="trigger.offsetWidth" class="popover-content">
               <div class="popover-item" (click)="this.onFilterValues()" *ngIf="this.isFilterable">Filter Values</div>
               <div class="popover-item-divider" *ngIf="this.isFilterable"></div>
-              <div class="popover-item" (click)="this.onSortChange(SORT_ASC)">
+              <div
```

Comment:
Are we also ensuring the divider lines match up with sorting turned both on and off. We would need to check with the other options - Filtering and Edit - on and off as well. A few combinations to test here.

Reply:
@jake-bassett done. used ng-containers to segregate a section and associated dividers.